tools/spectral/ipa/__tests__/__helpers__/testRule.js

```diff
@@ -0,0 +1,40 @@
+import * as fs from 'node:fs';
+import * as path from 'node:path';
+import { describe, expect, it } from '@jest/globals';
+import { Spectral, Document } from '@stoplight/spectral-core';
+import { httpAndFileResolver } from '@stoplight/spectral-ref-resolver';
+import { bundleAndLoadRuleset } from '@stoplight/spectral-ruleset-bundler/with-loader';
+
+const rulesetPath = path.join(__dirname, '../..', 'ipa-spectral.yaml');
+
+export default (ruleName, tests) => {
```

Comment:
Nit: Should we add this repo structure for unit tests to the spec?

Reply:
I can add it to the appendix 👍

Reply:
https://docs.google.com/document/d/1hcYKgnD7FUGII-3BP4TgiBc9rDcNyd2LMuzbxqFUl1Q/edit?tab=t.0#bookmark=id.irlwpe8w17gn